controls/apache_spec.rb

```diff
@@ -117,22 +137,16 @@
   end
 end
 
-control 'apache-07' do
```

Comment:
We probably should not rename the controls, but instead just add a new one. This way we won't break backwards compatibility.